test/common/index.js

```diff
@@ -938,6 +939,18 @@ function getPrintedStackTrace(stderr) {
   return result;
 }
 
+/**
+ * Check the exports of require(esm).
+ * TODO(joyeecheung): use it in all the test-require-module-* tests to minimize changes
+ * if/when we change the layout of the result returned by require(esm).
+ * @param {object} mod result returned by require()
+ * @param {object} expectation shape of expected namespace.
+ */
+function expectRequiredModule(mod, expectation) {
+  assert(isModuleNamespaceObject(mod));
```

Comment:
Should we add a assert message to here as well? For better understanding, and debugging?

Reply:
Do you have a suggestion? I think "mod must be a module namespace object" is already self explanatory. The object being checked is expected to be a module namespace object, otherwise the test fails, what else needs to be said about this?